### PR TITLE
Updated Estonian translation files

### PR DESCRIPTION
**Pull request by Rvcboy**
Translated everything again to make sure translations are up to date.

### CODE REVIEW

File: src/main/resources/messages/help_et.yml [hunk 1]
```diff
@@ -35,11 +35,11 @@ section:
 # Important: Put main commands like "authme" before their children (e.g. "authme.reload")
 commands:
     authme.register:
-        description: 'Registreeri mängija'
-        detailedDescription: 'Registreeri valitud mängija valitud parooliga.'
+        description: 'Register a player'
```
Comment (sgdc3) [inline src/main/resources/messages/help_et.yml]: ehm, what?

File: src/main/resources/messages/messages_et.yml [hunk 2]
```diff
@@ -5,152 +5,150 @@
...
-    disabled: '&cMängusisene registreerimine välja lülitatud!'
-    name_taken: '&cKasutajanimi juba registreeritud!'
-    register_request: '&3Palun registreeri käsklusega /register <Parool> <KinnitaParool>'
-    command_usage: '&cKasutus: /register <KinnitaParool> <KinnitaParool>'
-    reg_only: '&4Ainult registreeritud mängijad saavad liituda! Külasta veebilehte http://example.com, et ennast regsitreerida.'
-    success: '&2Edukalt registreeritud!'
-    kicked_admin_registered: 'Administraator registreeris sind, palun lahku korra serverist ja liitu tagasi.'
+  register_request: '&3Palun registreeri käsklusega: /register <parool> <KinnitaParool>'
+  command_usage: '&cKasutus: /register <parool> <KinnitaParool>'
+  reg_only: '&4Vaid registreeritud mängijad saavad liituda serveriga! Külasta http://näide.ee enda kasutaja registreerimiseks!'
```
Comment (sgdc3) [inline src/main/resources/messages/messages_et.yml]: Please keep the default url

File: src/main/resources/messages/help_et.yml (hunk 1 above)
Comment (Rvcboy) [reply src/main/resources/messages/help_et.yml]: Translated that part again or should I change it back to the English version?

**Comment by sgdc3** [comment]
Thank you ;)

**Pull request merged**